mappings/net/minecraft/client/render/entity/EntityRendererArrows.mapping

```diff
@@ -0,0 +1,7 @@
+CLASS none/bvg net/minecraft/client/render/entity/EntityRendererArrows
```

Comment:
That's confusing then, how about `StuckArrows`? I guess it could *still* be confused for allows stuck in the ground or so but at least it's a step in the right direction, imo.